javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,23 @@
+var fullName, isCool, totalPets;
+var unknown;
+var never;
+
+fullName = 'Joseph Nguyen';
+isCool = true;
+totalPets = 0;
```

Comment:
You don't need to declare your variables before giving them value, you can declare them and assign value on the same line.

Reply:
I remember reading in the prep course that it was generally good practice in JavaScript to make all of the variable declarations at the top, and this was the practice used in a similar language I learned (Verilog).  Is this not the case?